### ISSUE TITLE
Crash while converting

### ISSUE THREAD

**Issue by dadodasyra** [HiveGamesOSS/Chunker#45]
### Chunker Version

cli-1.1.2

### Platform / Operating System

Debian 11

### Issue Description

# Input world
1.20.1 "Modded" Java (see more details in description)
Same setup as #36 but with a different map (same generation, not same seed).

I've got this error after a little bit without further notice.
```
82.95%
java.lang.IllegalArgumentException: Could not read list with length 8319
        at com.hivemc.chunker.nbt.tags.collection.ListTag.decodeValue(ListTag.java:146)
        at com.hivemc.chunker.nbt.tags.collection.CompoundTag.decodeValue(CompoundTag.java:102)
        at com.hivemc.chunker.nbt.tags.Tag.decodeNamed(Tag.java:568)
        at com.hivemc.chunker.nbt.tags.Tag.readZLibJavaNBT(Tag.java:483)
        at com.hivemc.chunker.conversion.encoding.java.base.reader.util.MCAReader.decompressColumn(MCAReader.java:114)
        at com.hivemc.chunker.conversion.encoding.java.base.reader.util.MCAReader.lambda$readColumn$0(MCAReader.java:97)
        at com.hivemc.chunker.scheduling.function.ThrowableSupplier.invoke(ThrowableSupplier.java:20)
        at com.hivemc.chunker.scheduling.task.WrappedTask.apply(WrappedTask.java:84)
        at com.hivemc.chunker.scheduling.task.WrappedTask.get(WrappedTask.java:77)
        at com.hivemc.chunker.scheduling.task.executor.TaskExecutorSupplier.run(TaskExecutorSupplier.java:42)
        at com.hivemc.chunker.scheduling.task.executor.TaskExecutor.threadLoop(TaskExecutor.java:178)
        at java.base/java.lang.Thread.run(Thread.java:833)
Failed with exception
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Could not read list with length 8319
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
        at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:936)
        at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
        at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)
        at com.hivemc.chunker.scheduling.task.Environment.cancel(Environment.java:85)
        at com.hivemc.chunker.conversion.WorldConverter.cancel(WorldConverter.java:609)
        at com.hivemc.chunker.conversion.WorldConverter.logFatalException(WorldConverter.java:441)
        at com.hivemc.chunker.scheduling.task.executor.TaskExecutor.handleException(TaskExecutor.java:109)
        at com.hivemc.chunker.scheduling.task.WrappedTask.lambda$setupFuture$0(WrappedTask.java:57)
        at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
        at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
        at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)
        at com.hivemc.chunker.scheduling.task.executor.TaskExecutorSupplier.run(TaskExecutorSupplier.java:44)
        at com.hivemc.chunker.scheduling.task.executor.TaskExecutor.threadLoop(TaskExecutor.java:178)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: Could not read list with length 8319
        at com.hivemc.chunker.nbt.tags.collection.ListTag.decodeValue(ListTag.java:146)
        at com.hivemc.chunker.nbt.tags.collection.CompoundTag.decodeValue(CompoundTag.java:102)
        at com.hivemc.chunker.nbt.tags.Tag.decodeNamed(Tag.java:568)
        at com.hivemc.chunker.nbt.tags.Tag.readZLibJavaNBT(Tag.java:483)
        at com.hivemc.chunker.conversion.encoding.java.base.reader.util.MCAReader.decompressColumn(MCAReader.java:114)
        at com.hivemc.chunker.conversion.encoding.java.base.reader.util.MCAReader.lambda$readColumn$0(MCAReader.java:97)
        at com.hivemc.chunker.scheduling.function.ThrowableSupplier.invoke(ThrowableSupplier.java:20)
        at com.hivemc.chunker.scheduling.task.WrappedTask.apply(WrappedTask.java:84)
        at com.hivemc.chunker.scheduling.task.WrappedTask.get(WrappedTask.java:77)
        at com.hivemc.chunker.scheduling.task.executor.TaskExecutorSupplier.run(TaskExecutorSupplier.java:42)
        ... 2 more
root@bedrock:~/convert#
```

**Comment by FormallyMyles** [comment]
If you're able to attach or email a region file (.mca) or the map to `chunker-public@hivemc.com` we can look further. There is basic protection against really big lists (8192 is the limit at the moment), if we can verify that there's nothing wrong with it being this size we can look to up the limit.

It can sometimes be a sign of corrupt data but it's one of those things where you have to inspect it to work that out.

**Comment by dadodasyra** [comment]
> If you're able to attach or email a region file (.mca) or the map to `chunker-public@hivemc.com` we can look further. There is basic protection against really big lists (8192 is the limit at the moment), if we can verify that there's nothing wrong with it being this size we can look to up the limit.
> 
> It can sometimes be a sign of corrupt data but it's one of those things where you have to inspect it to work that out.

I don't really know how I can figure out what specific region file is concerned by this crash. 
Here's the full input map (22GB unzipped): http://drive.histeria.fr/f/534cf82a31014f738786/

**Comment by FormallyMyles** [comment]
I had a quick look into this and seems related to a chunk with lots of `fluid_ticks` which we can up the limit for the ListTag since it's hitting the 8192 limit we have.

**Comment by FormallyMyles** [comment]
This should be fixed in our latest release 1.1.3 - https://github.com/HiveGamesOSS/Chunker/releases, let us know if there's any issues :)
